config/health_check.php

```diff
@@ -2,10 +2,12 @@
 
 use Saritasa\LaravelHealthCheck\Checkers\DatabaseHealthChecker;
 use Saritasa\LaravelHealthCheck\Checkers\RedisHealthChecker;
+use Saritasa\LaravelHealthCheck\Checkers\S3HealthChecker;
 
 return [
     'checkers' => [
         'database' => DatabaseHealthChecker::class,
         'redis' => RedisHealthChecker::class,
+        's3' => S3HealthChecker::class,
```

Comment:
I think better to rename slug and checker name to `CloudFileStorage` , because in 99% it will be s3, but in laravel conception it's additional level of abstraction. So to avoid misunderstanding in future better to keep names with it real purpose. @populov what do you think about it ?

Reply:
Good catch, on one side.
On other hand, then you need to explain to a new DevOps, that "cloud = S3". So we can just leave it S3, because I bet 99% you don't want to explain to anyone, who 99% doesn't read documentation, how it works :smile: